scico/flax/train/train.py

```diff
@@ -0,0 +1,1151 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2022 by SCICO Developers
+# All rights reserved. BSD 3-clause License.
+# This file is part of the SCICO package. Details of the copyright and
+# user license can be found in the 'LICENSE' file distributed with the
+# package.
+
+"""Utilities for training Flax models.
```

Comment:
This module has more than 1000 lines, which can make maintenance difficult. Is there any logical structure that could be used to break it into smaller modules?

Reply:
Train functionality has been restructured.